Gemfile

```diff
@@ -17,11 +17,13 @@ gem 'govuk_design_system_formbuilder', '~> 1.1'
 gem 'haml-rails', '~> 2.0.1'
 gem 'json-schema', '~> 2.8', '>= 2.8.1'
 gem 'json_api_client', '~> 1.16', '>= 1.16.1'
+gem 'oauth2', '~> 1.4.4'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.3'
 gem 'rails', '~> 6.0.2', '>= 6.0.2.1'
 gem 'redis', '~> 4.1.3'
 gem 'sidekiq', '~> 5.2.7'
+gem 'vcr'
```

Comment:
minor issue: should this be within the test group?

Reply:
👍 yes probably, don't know why i put it there (brain malfunction or otherwise?!)